WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -119,9 +118,9 @@ class LoginTests: XCTestCase {
         XCTAssert(WelcomeScreen().isLoaded())
     }
 
-    // Unified email login fail
+    // Unified WordPress.com email login fail due to incorrect password
```

Comment:
What do you think of the following?

```suggestion
    // Unified WordPress.com email login failure due to incorrect password
```